### PR TITLE
DT-1408: ✅ Add in testing of DPS search

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: backend/contextProperties.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ const copyNamedHeaders = (headerNames, srcHeaders) =>
     if (srcHeaders[name]) {
       return {
         ...previous,
-        [name]: parseInt(srcHeaders[name], 10),
+        [name]: srcHeaders[name],
```
Comment (petergphillips) [inline backend/contextProperties.js:37]: from previous PR - would fail if header isn't a number

File: backend/routes/searchDpsUserRouter.js [hunk 2]
```diff
@@ -10,10 +10,9 @@ const controller = ({ prisonApi, logError }) => {
     const hasAdminRole = Boolean(context && context.user && context.user.maintainAccessAdmin)
 
     contextProperties.setRequestPagination(context, { offset, size })
-    if (hasAdminRole) {
-      return prisonApi.userSearchAdmin(context, { nameFilter, roleFilter: roleCode })
-    }
-    return prisonApi.userSearch(context, { nameFilter, roleFilter: roleCode })
+    return hasAdminRole
```
Comment (petergphillips) [inline backend/routes/searchDpsUserRouter.js:13]: from previous PR - remove `if`

File: tsconfig.json [hunk 3]
```diff
@@ -15,6 +15,6 @@
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true
   },
-  "exclude": ["node_modules", "assets", "integration-tests", "build", "dist", "static", "config"],
+  "exclude": ["node_modules", "assets", "integration-tests", "build", "dist", "static", "config", "manage-accounts-specs"],
```
Comment (andrewrlee) [inline tsconfig.json:18]: is this necessary?

**Review by andrewrlee — APPROVED** [review]
LGTM

File: tsconfig.json (hunk 3 above)
Comment (petergphillips) [reply tsconfig.json:18]: yes, because the groovy tests contain a `build/reports` directory that contains javascript that was being validated

**Pull request merged**